esx_service/utils/vmdk_utils.py

```diff
@@ -44,18 +46,23 @@ def get_datastores():
     #  We are connected to ESX so childEntity[0] is current DC/Host
     ds_objects = \
       si.content.rootFolder.childEntity[0].datastoreFolder.childEntity
-    datastores = [(d.info.name,
+    datastores = [[d.info.name,
                    os.path.split(d.info.url)[1],
-                   os.path.join(d.info.url, 'dockvols'))
+                   os.path.join(d.info.url, 'dockvols'), True]
                   for d in ds_objects]
+    for d in ds_objects:
+       ds = [x for x in datastores if x[0] == d.info.name]
+       if d.summary.type == 'VMFS':
+          ds[0][3] = d.info.vmfs.local
```

Comment:
I am not sure why "VMFS "indicates "local". What if it's just an array LUN exposed via iSCSI and formatted as VMFS (?